docs/api/guides/guide-to-creating-an-endpoint.md

```diff
@@ -275,3 +279,4 @@ var eventModels = map[KeyType]eventModel{
 
 
 If you'd like to learn more about event triggers, you can find more details [here](https://github.com/transcom/mymove-docs/blob/720592c63db4bffe402a801417f7c14772573c28/docs/dev/contributing/backend/How-to-Add-an-Event-Trigger.md).
+
```

Comment:
Sometimes I wonder if we should standardize on adding or not adding a newline at the end of the file.